python/v1/0/statement_pb2.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: v1.0/statement.proto
+
+from google.protobuf import descriptor as _descriptor
+from google.protobuf import message as _message
+from google.protobuf import reflection as _reflection
+from google.protobuf import symbol_database as _symbol_database
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+from google.protobuf import struct_pb2 as google_dot_protobuf_dot_struct__pb2
+
+
+DESCRIPTOR = _descriptor.FileDescriptor(
+  name='v1.0/statement.proto',
+  package='io.in_toto.attestation.v1_0',
+  syntax='proto3',
+  serialized_options=b'Z+github.com/in-toto/attestation/go/spec/v1.0',
```

Comment:
As someone who isn't very experienced with protobuf, is this supposed to point into the go package? Is this some default carrying over?

Reply:
FWIW all this code is autogenerated by protoc, so who knows?